src/input.cpp

```diff
@@ -882,18 +884,30 @@ void input_context::display_help()
         ctxt.register_action( "HELP_KEYBINDINGS" );
     }
 
+    std::set<long> character_blacklist = std::set<long>();
```

Comment:
The set can be initialized with an initializer list: `std::set<...> foo = { 'a', 'b', 'c' };` and than you can even declare it as const.